pkg/dns/aws/dns.go

```diff
@@ -37,7 +37,6 @@ import (
 
 const (
 	ProviderSpecificEvaluateTargetHealth       = "aws/evaluate-target-health"
-	ProviderSpecificWeight                     = "aws/weight"
 	ProviderSpecificRegion                     = "aws/region"
 	ProviderSpecificFailover                   = "aws/failover"
 	ProviderSpecificGeolocationContinentCode   = "aws/geolocation-continent-code"
```

Comment:
All of these will eventually be moved in favour of generic options that will be used by all providers. For now i have only moved the weight since it's the only one currently used by mgc.